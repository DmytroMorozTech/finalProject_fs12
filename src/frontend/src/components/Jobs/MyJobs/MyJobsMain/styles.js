import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  container: {
    borderRadius: theme.shape.medium,
    backgroundColor: theme.palette.common.white,
    paddingTop: theme.spacing(3)
  },

  title: {
    fontSize: theme.typography.h3.fontSize,
    color: theme.palette.grey[900],
    paddingLeft: theme.spacing(5)
  },

  btn: {
    backgroundColor: theme.palette.success,
    color: theme.palette.common.white,
    marginLeft: theme.spacing(3)
  },

  line: {
    width: '100%',
    height: '0.5px',
    border: '0',
    margin: '0 auto',
    backgroundColor: theme.palette.grey[300]
  }
}))
