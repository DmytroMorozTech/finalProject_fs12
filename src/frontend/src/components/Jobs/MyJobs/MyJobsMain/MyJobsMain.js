import styles from './styles'
import React from 'react'
import SharedButton from '../../../../shared/SharedButton/SharedButton'
import SavedVacancy from './SavedVacancy/SavedVacancy'

function MyJobsMain () {
  const classes = styles()

  return (
    <div className={classes.container}>
      <div className={classes.title}>
        My Jobs
      </div>
      <div className={classes.btn}>
        <SharedButton title="Saved"/>
      </div>
      <hr className={classes.line}/>
      <div>
        <SavedVacancy/>
      </div>
    </div>
  )
}

export default MyJobsMain
