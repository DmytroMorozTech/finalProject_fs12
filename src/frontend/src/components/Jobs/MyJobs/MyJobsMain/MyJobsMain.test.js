import { shallow } from 'enzyme'
import React from 'react'
import MyJobsMain from './MyJobsMain'

jest.mock('./styles', () => () => ({}))

const setUp = () => shallow(<MyJobsMain/>)

describe('should render MyJobsMain component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
