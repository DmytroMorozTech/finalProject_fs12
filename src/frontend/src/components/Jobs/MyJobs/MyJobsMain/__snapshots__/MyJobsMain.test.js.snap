// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MyJobsMain component to match snapshot 1`] = `
<div>
  <div>
    My Jobs
  </div>
  <div>
    <SharedButton
      title="Saved"
    />
  </div>
  <hr />
  <div>
    <SavedVacancy />
  </div>
</div>
`;
